x/dispensation/Readme-Distrubute.md

```diff
@@ -16,17 +14,19 @@
 ```go
 package records
 type DistributionType int64
-
+const DistributionTypeUnknown DistributionType = 0
 const Airdrop DistributionType = 1
 const LiquidityMining DistributionType = 2
 const ValidatorSubsidy DistributionType = 3
 
 type Distribution struct {
-    DistributionType DistributionType `json:"distribution_type"`
-    DistributionName string           `json:"distribution_name"`
+	DistributionType DistributionType `json:"distribution_type"`
+	DistributionName string           `json:"distribution_name"`
+	Runner           sdk.AccAddress   `json:"runner"`
 }
 ```
-This is stored in the keeper with the key DistributionType_DistributionName for historical records. Therefore, the combination of type and name needs to be unique
+This is stored in the keeper with the key DistributionName_DistributionType for historical records. 
+The Distribution name is BlockHeight_Distributor ,Therefore, the combination BlockHeight_Distributor_DistributionType needs to be unique
 
```

Comment:
The Distribution name is BlockHeight_Distributor, therefore, the combination BlockHeight_Distributor_DistributionType needs to be unique